test/console/control_flow_commands_test.rb

```diff
@@ -21,7 +21,9 @@ def program
         10|
         11| s = Student.new("John")
         12| s.name
-        13| "foo"
+        13| puts "foo"
+        14| puts "bar"
+        15| "baz"
```

Comment:
Added this to the test to ensure that `next <number>` functionality is preserved when stepping over multiple lines at the same stack depth.